<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSerialPort</class>
 <widget class="QDialog" name="DialogSerialPort">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelPort">
     <property name="text">
      <string> 串    口：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxPort"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelBaud">
     <property name="text">
      <string>波  特  率：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBoxBaud"/>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnPortConnect">
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnPortRefresh">
     <property name="text">
      <string>刷新</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
